cas-client-core/src/main/java/org/jasig/cas/client/util/CommonUtils.java

```diff
@@ -438,8 +439,12 @@ public static String getResponseFromServer(final URL constructedUrl, final HttpU
             }
 
             return builder.toString();
+        } catch (final SSLException e) {
+            LOGGER.error("SSL error getting response from host: {} : Error Message: {}", constructedUrl.getHost(), e.getMessage(), e);
+            throw new RuntimeException(e);        	
         } catch (final Exception e) {
-            LOGGER.error(e.getMessage(), e);
+            LOGGER.error("Error getting response from host: {} with path: {} and protocol: {}: Error Message: {}",
+            		constructedUrl.getHost(), constructedUrl.getPath(), constructedUrl.getProtocol(), e.getMessage(), e);
```

Comment:
I will update pull request later, this doesn't handle someone passing in a null constuctedUrl which would cause an NPE and there would be another NPE in the Logger statement in the catch block.